traffic_portal/app/src/common/api/DeliveryServiceSslKeysService.js

```diff
@@ -17,10 +17,10 @@
  * under the License.
  */
 
-var DeliveryServiceSslKeysService = function($http, $q, locationUtils, messageModel, ENV) {
+var DeliveryServiceSslKeysService = function($http, locationUtils, messageModel, ENV) {
 	this.generateSslKeys = function(deliveryService, sslKeys, generateSslKeyForm) {
 		if (sslKeys.hasOwnProperty('version')){
-			generateSslKeyForm.version = parseInt(sslKeys.version) + 1;
+			generateSslKeyForm.version = parseInt(sslKeys.version, 10) + 1;
```

Comment:
seems unrelated to restangular

Reply:
It is, but [using `parseInt` without passing an explicit radix is dangerous](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt#Octal_interpretations_with_no_radix) so I wanted to fix it asap.